app/assets/js/rtMedia.backbone.js

```diff
@@ -1066,9 +1102,30 @@ jQuery( document ).ready( function( $ ) {
 			}
 		}, 100 );
 
-		if ( $( '#whats-new-options' ).length > 0 && $( '#whats-new-form .rtmedia-uploader-div' ).length > 0 ) {
-			$( '#whats-new-options' ).append( $( '#whats-new-form .rtmedia-uploader-div' ) );
+		/**
+		 * Appends rtMedia Uploader option below form content section
+		 * Issue 243
+		 */
+		if( bp_template_pack && 'legacy' !== bp_template_pack ) {
+
+			let form_ref        = jQuery( '#whats-new-form' );
+			let rt_uploader_div = jQuery( '.rtmedia-uploader-div' );
+
+			if ( 0 < jQuery( '.activity-update-form' ).length && 0 < form_ref.length && 0 < rt_uploader_div.length ) {
+				$( form_ref ).append( rt_uploader_div );
+				rt_uploader_div.hide();
+			}
+
+		} else {
+
+			if ( $( '#whats-new-options' ).length > 0 && $( '#whats-new-form .rtmedia-uploader-div' ).length > 0 ) {
+
+				$( '#whats-new-options' ).append( $( '#whats-new-form .rtmedia-uploader-div' ) );
```

Comment:
Instead of doing `$( '#whats-new-options' )` several times, take 1 variable having this reference and perform checks / appends on those to prevent redundant DOM queries.